homeassistant/components/logger/__init__.py

```diff
@@ -13,25 +14,14 @@
 SERVICE_SET_DEFAULT_LEVEL = "set_default_level"
 SERVICE_SET_LEVEL = "set_level"
 
-HIGHEST_LOG_LEVEL = logging.CRITICAL
-
-LOGSEVERITY = {
-    "CRITICAL": 50,
-    "FATAL": 50,
-    "ERROR": 40,
-    "WARNING": 30,
-    "WARN": 30,
-    "INFO": 20,
-    "DEBUG": 10,
-    "NOTSET": 0,
-}
+DEFAULT_LOGSEVERITY = "DEBUG"
```

Comment:
Would this be a good time to change this default to INFO? It's way too verbose when we set it to DEBUG

Reply:
If we set this to INFO it will probably break `caplog` and other tests that check for logger messages since it will lower the default log level (which is effectively a global) below what is expected.